test/FunctionalTests/Microsoft.OData.Core.Tests/UriParser/Binders/FunctionCallBinderTests.cs

```diff
@@ -1326,6 +1256,36 @@ public void CannotCallFunctionInOpenTypeSpace()
             bindOpenFunction.ShouldThrow<ODataException>().WithMessage(ODataErrorStrings.FunctionCallBinder_CallingFunctionOnOpenProperty("Fully.Qualified.Namespace.FindMyOwner"));
         }
 
+        // Make sure function signatures are first searched in Custom Functions cache, and then in BuiltIn cache
+        [Fact]
+        public void GetUriFunction_SearchForCustomFunctionBeforeBuiltIn()

```

Comment:
For most custom they don't use custom functions. so it should be better to search built-in first, then custom.